pelicun/tests/basic/test_base.py

```diff
@@ -667,21 +657,21 @@ def test_str2bool():
     assert base.str2bool('f') is False
     assert base.str2bool('1') is True
     assert base.str2bool('0') is False
-    assert base.str2bool(True) is True
-    assert base.str2bool(False) is False
+    assert base.str2bool(v=True) is True
+    assert base.str2bool(v=False) is False
```

Comment:
See earlier commend on the "boolean trap". Here we have an example of a case where it wouldn't really be ambiguous, but rather than silencing the warning we can just pass the value as a keyword argument for consistency.

Reply:
I see, okay.